code-review.md

```diff
@@ -0,0 +1,26 @@
+# Code Review
+
+A peer review of source code. It is intended to find and fix mistakes overlooked in the initial development phase, improving both the overall quality of software and the developers' skills.
+
+## Beginning Code Review
+
+* Read [Feature branch code reviews](http://smartbear.com/SmartBear/media/pdfs/WP-CC-11-Best-Practices-of-Peer-Code-Review.pdf)
+* Read [Best practices of peer code review](http://goo.gl/A8Boc)
```

Comment:
expand to: http://smartbear.com/SmartBear/media/pdfs/WP-CC-11-Best-Practices-of-Peer-Code-Review.pdf